src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+        <table>
+            <thead>
+                <tr>
+                    <th>Data</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>1</td>
+                    <td>
+                        Connected = 1
+                        <br></br>
+                        Disconnected = 0
+                    </td>
+                </tr>
+                <tr>
+                    <td>2-3</td>
+                    <td>External Device ID</td>
+                </tr>
+                <tr>
+                    <td>4-5</td>
+                    <td>Connection flags</td>
+                </tr>
+            </tbody>
+        </table>
+        <table>
+            <thead>
+                <tr>
+                    <th>Bits</th>
+                    <th>Connection flags</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Reserved</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Binary Data Packet Wrapping</td>
+                </tr>
+                <tr>
+                    <td>2 - 15</td>
+                    <td>Reserved</td>
+                </tr>
+            </tbody>
+        </table>
+        <p>Binary Data Packet Wrapping: 0: The passthrough data from MyGeotab will be passed to the external device without modification. 1: The passthrough data from MyGeotab will be wrapped in a serial protocol before being sent to the external device. Note: If sending large payloads of variable sizes, it is recommended to use the binary wrapping flag to allow the device to distinguish and accommodate different packet sizes. The device should implement support for both 0x23 and 0x25 message formats as the GO will dynamically select which one to use based on the amount of data within each packet received from MyGeotab. The maximum packet size currently supported is 1000 bytes.</p>
+        <p>For payloads with a length of 0 - 255 bytes, this format is used: | | Bytes | Position | | — | — | — | | STX (0x02) | 1 | 0 | | Message Type = 0x23 | 1 | 1 | | Message Body Length = x | 1 | 2 | | Binary Data | x | 3 | | Checksum | 2 | 3+x | | ETX (0x03) | 1 | 5+x |</p>
+        <p>For payloads with a length of 256 - 1000 bytes, this format is used: | | Bytes | Position | | — | — | — | | STX (0x02) | 1 | 0 | | Message Type = 0x25 | 1 | 1 | | Message Body Length = x | 2 | 2 | | Binary Data | x | 4 | | Checksum | 2 | 4+x | | ETX (0x03) | 1 | 6+x |</p>
+        <p>More details on the checksum can be found here: <a href="/sdk/hardware/addon-protocols/rs232-usb/#checksum">Add-On Protocol - RS232 &amp; USB</a></p>
+        <h4>Log Type: 2 (GenericFaultRecord)</h4>
+        <p>Typically used to log a fault condition that needs to be escalated to a supervisor for human intervention.</p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on